.github/workflows/compile-queries.yml

```diff
@@ -7,6 +7,11 @@ on:
       - "rc/*"
       - "codeql-cli-*"
   pull_request:
+    paths:
+      - '*.ql'
+      - '*.qll'
+      - '**/qlpack.yml'
+      - '*.dbscheme'
```

Comment:
Don't all the lines need a `**/`?

Reply:
Indeed, thanks for catch. I updated according to https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-including-paths